BasicSample/Assets/HandTracking/Scripts/ToggleMRTKHandVisualization.cs

```diff
@@ -22,7 +22,7 @@ public class ToggleMRTKHandVisualization : MonoBehaviour
         /// </summary>
         private void Start()
         {
-            MixedRealityInputSystemProfile inputSystemProfile = CoreServices.InputSystem?.InputSystemProfile;
+            MixedRealityInputSystemProfile inputSystemProfile = MrtkCoreServices.InputSystem?.InputSystemProfile;
```

Comment:
Another way I just found to fix this: uninstall the Version Control package in UPM. I don't think we use it at all, and it's what introduces this conflict. It's in the manifest.json as `"com.unity.collab-proxy": "1.3.9",` and shows up in the UI as Version Control.

Reply:
Good idea.  Took your suggestion and updated to latest iteration.